cypress/support/step_definitions/fireboltCalls.js

```diff
@@ -365,3 +365,125 @@ Given('device is rebooted', () => {
     throw new Error(CONSTANTS.STEP_IMPLEMENTATION_MISSING);
   });
 });
+
+/**
+ * @module fireboltCalls
+ * @function And 3rd party '(.+)' app is dismissed
+ * @description To dismiss the launched app
+ * @param {String} app - app name.
+ * @example
+ * And 3rd party 'firebolt' app is dismissed
+ */
+Given(/3rd party '(.+)' app is dismissed$/, async (appType) => {
+  const appId = Cypress.env(CONSTANTS.RUNTIME).appId;
+  let KeyPressSequence;
+  if (
+    Cypress.env(CONSTANTS.RUNTIME) &&
+    Cypress.env(CONSTANTS.RUNTIME).intent &&
+    Cypress.env(CONSTANTS.RUNTIME).intent.keyPressSequence
+  ) {
+    KeyPressSequence = Cypress.env(CONSTANTS.RUNTIME).intent.keyPressSequence;
+  } else if (
+    Cypress.env('app_metadata') &&
+    Cypress.env('app_metadata')[appId] &&
+    Cypress.env('app_metadata')[appId].defaultKeyPressSequence
+  ) {
+    KeyPressSequence = Cypress.env('app_metadata')[appId].defaultKeyPressSequence;
+  } else if (Cypress.env('app_metadata') && Cypress.env('app_metadata').defaultKeyPressSequence) {
+    KeyPressSequence = Cypress.env('app_metadata').defaultKeyPressSequence;
+  } else {
+    throw new Error('No KeyPressSequence ');
+  }
```

Comment:
Log should be descriptive. how to fix if there is no kepress sequence

Reply:
updated the log

Reply:
app_metadata should be in constant for deeplink and dismiss

Reply:
throw new Error(`Expected KeyPressSequence was not found for ${appId} in app_metadata.json. More details on app_metadata present <link to app metadatajson template and readme>`); - 

add a template for app_metadata.json in FCS with all dummy values

Reply:
added a template app_metadata.json